katas/content/marking_oracles/periodic/Placeholder.qs

```diff
@@ -0,0 +1,6 @@
+namespace Kata {
+    operation Oracle_Periodic (x : Qubit[], y : Qubit) : Unit is Adj + Ctl {
+        // Implement your solution here...
+
+    }    
```

Comment:
Ah, in this one we expect them to reuse the code from the previous exercise, but we don't have a way to do dependencies directly. What we do in such cases is say "// You might find this helper operation from an earlier task useful." and copy-paste our expected solution for Oracle_PeriodicGivenPeriod below (see https://github.com/microsoft/qsharp/blob/main/katas/content/qec_shor/shor_encode/Placeholder.qs for an example). Let's do it here